### DISCUSSION TITLE
Version not available for 2.2404.x, using latest as fallback - Message starting WppConnect

### DISCUSSION THREAD

**Discussion by vqmacedo** [wppconnect-team/wppconnect#2156] [General]
info:     
  _       ______  ____  ______                            __ 
  | |     / / __ \/ __ \/ ____/___  ____  ____  ___  _____/ /_
  | | /| / / /_/ / /_/ / /   / __ \/ __ \/ __ \/ _ \/ ___/ __/
  | |/ |/ / ____/ ____/ /___/ /_/ / / / / / / /  __/ /__/ /_  
  |__/|__/_/   /_/    \____/\____/_/ /_/_/ /_/\___/\___/\__/
info:     [F84B20AB-958B-48CC-A864-3FDFE5C04CB7:browser] Using browser folder './tokens/F84B20AB-958B-48CC-A864-3FDFE5C04CB7'
info:     [F84B20AB-958B-48CC-A864-3FDFE5C04CB7:browser] Initializing browser...
http:     [F84B20AB-958B-48CC-A864-3FDFE5C04CB7:browser] checking headless...
http:     [F84B20AB-958B-48CC-A864-3FDFE5C04CB7:browser] headless option is active, browser hidden
info:     [F84B20AB-958B-48CC-A864-3FDFE5C04CB7:client] Initializing...
**verbose:  [F84B20AB-958B-48CC-A864-3FDFE5C04CB7:client] Setting WhatsApp WEB version to 2.2404.x
error:    [F84B20AB-958B-48CC-A864-3FDFE5C04CB7:client] Version not available for 2.2404.x, using latest as fallback**
http:     [F84B20AB-958B-48CC-A864-3FDFE5C04CB7:client] Waiting page load
verbose:  [F84B20AB-958B-48CC-A864-3FDFE5C04CB7:client] Loading WhatsApp WEB
verbose:  [F84B20AB-958B-48CC-A864-3FDFE5C04CB7:client] WhatsApp WEB loaded
verbose:  [F84B20AB-958B-48CC-A864-3FDFE5C04CB7:page] Page loaded
verbose:  [F84B20AB-958B-48CC-A864-3FDFE5C04CB7:client] Injecting wapi.js
^CSIGINT
cleanup

**Comment by icleitoncosta** [comment]
Update your projetct.
